web/src/components/AppStoreBanner.tsx

```diff
@@ -0,0 +1,111 @@
+import { EmapsIcon } from 'icons/emapsIcon';
+import { useAtom } from 'jotai';
+import { atomWithStorage } from 'jotai/utils';
+import { X } from 'lucide-react';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import trackEvent from 'utils/analytics';
+import { TrackEvent } from 'utils/constants';
+
+import { isAndroid, isIphone } from '../features/weather-layers/wind-layer/util';
+import { Button } from './Button';
+
+export const appStoreDismissedAtom = atomWithStorage('isAppBannerDismissed', false);
+
+export enum AppStoreURLs {
+  APPLE = 'https://apps.apple.com/us/app/electricity-maps/id1224594248',
+  GOOGLE = 'https://play.google.com/store/apps/details?id=com.tmrow.electricitymap&hl=en',
+}
+
+export interface AppStoreBannerProps {
+  CloseButton?({ onClose }: DefaultCloseButtonProps): React.ReactNode;
+}
+
+export function AppStoreBanner({
+  CloseButton = DefaultCloseButton,
+}: AppStoreBannerProps) {
+  const { appStoreUrl, closeBanner } = useAppStoreBannerState();
+  const { t } = useTranslation();
+
+  return (
+    appStoreUrl && (
+      <div
+        role="banner"
+        aria-live="polite"
+        className="sticky z-50 flex h-14 min-h-14 w-full items-center border-b border-solid border-neutral-300 bg-neutral-100 px-3"
+      >
+        <CloseButton onClose={closeBanner} />
+        <div className="flex flex-grow gap-2">
+          <div className="items-center justify-center self-center rounded-md border border-neutral-200 bg-white">
+            <EmapsIcon size={40} />
+          </div>
+          <div className="content-center text-neutral-600">
+            <h3>Electricity Maps</h3>
+            <p className="text-xs">{t('app-banner.description')}</p>
+          </div>
+        </div>
+        <Button
+          size="md"
+          backgroundClasses={'h-9'}
+          href={appStoreUrl}
+          onClick={trackCTAClick}
+        >
+          {t('app-banner.cta')}
+        </Button>
+      </div>
```

Comment:
Should this also have a dark mode?

Looks a bit bright when everything else is dark 😎

<img width="478" alt="image" src="https://github.com/user-attachments/assets/ed72c38f-17dc-40e5-99e2-7c65f8a08c36">

Reply:
100%, thank you!